src/manager.cpp

```diff
@@ -120,6 +121,35 @@ sdbusplus::async::task<> Manager::startSyncEvents()
     co_return;
 }
 
+// NOLINTNEXTLINE
+void Manager::syncData(const config::DataSyncConfig& dataSyncCfg)
+{
+    using namespace std::string_literals;
+    //  TODO:
+    //  To be tuned with later commits.
+    std::string syncCmd{"rsync --archive --compress"};
+
+    // Add source data path
+    syncCmd.append(" "s + dataSyncCfg._path);
+
+#ifdef UNIT_TEST
+    syncCmd.append(" "s);
+#else
+    // TODO Support for remote (i,e sibling BMC) copying needs to be added.
+    syncCmd.append(" "s);
```

Comment:
I would prefer to fail while attempting to do remote copy (sibling BMC copy) so we could avoid adding the space here so that it will fail and we will see trace that you added in the line number 149).

For example:
```
/src/path/dest/path or /src/path/src/path and command will fail
```